### ISSUE TITLE
Accessibility: Hidden Inputs should have aria-hidden="true"

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#5846]
If you run Google Lighthouse on some components like SelectCheckBoxMenu.  its hidden inputs are being found by ScreenReaders when they should be `aria-hidden="true"` or should use `aria-label` if the label is known.

![image](https://user-images.githubusercontent.com/4399574/80657169-ec052b80-8a50-11ea-9214-dfe305e50ea2.png)

Components:
- [x] AutoComplete
- [x] SelectCheckBoxMenu
- [x] SelectOneMenu (Editor)
- [x] SelectBooleanButton
- [x] SelectOneButton
- [x] SelectOneListBox
- [x] Picklist
- [x] SelectManyButton
- [x] SelectManyMenu
- [x] SplitButton
- [x] Tree
